src/components/admin/lib/__tests__/article-validators.test.js

```diff
@@ -0,0 +1,19 @@
+import { checkForHttp, checkForAbsoluteUrlRegex } from '../article-validators';
+
+describe('Finds http/https in url', () => {
+  it('Returns true if it finds an http (not s) link ', () => {
+    expect(checkForHttp('https://www.google.com')).toEqual(true); // if the function returns true, the url is correct
+    expect(checkForHttp('http://www.google.com')).toEqual(false);
+    expect(checkForHttp('http')).toEqual(false);
```

Comment:
The think checkforHttp originally checked for functions that have http(not s), meaning that the first  expect would be correct but the second would be incorrect, as if is missing the s.

Reply:
Yeah, I completely agree with that, but it's more if we're thinking about the actual purpose, which is finding fully qualified urls, and since none of these are that it should be okay right. 

Also just thought, we should probably also have some testcases that embed the different urls inside some text, so just write some dummy data on each side, just a few words should be fine. Use `lorem ipsum` or whatever you want to put there